mailbox/elasticsearch/src/main/java/org/apache/james/mailbox/elasticsearch/events/ElasticSearchListeningMessageSearchIndex.java

```diff
@@ -108,10 +108,15 @@ public void add(MailboxSession session, Mailbox mailbox, MailboxMessage message)
         try {
             indexer.indexMessage(indexIdFor(mailbox, message.getUid()), messageToElasticSearchJson.convertToJson(message, ImmutableList.of(session.getUser())));
         } catch (Exception e) {
-            LOGGER.error("Error when indexing message " + message.getUid(), e);
+            try {
```

Comment:
I don't understand why we will reindex again when error. Do we ensure that the error because of no-attachment?